mir-module/src/main/resources/config/mir/messages_de.properties

```diff
@@ -228,11 +228,13 @@ mir.access                                      = Zugriffsberechtigung
 mir.accesskey                                   = Zugriffsschl\u00FCssel
 mir.accesskey.manage                            = Zugriffsschl\u00FCssel verwalten
 mir.accesskey.required                          = Bitte geben Sie einen Zugriffsschl\u00FCssel ein.
-mir.accesskey.setInfo                           = Gegebenenfalls besteht die M\u00F6glichkeit mit einem Zugriffsschl\u00FCssel \u00FCber folgenden <a href="{0}">Link</a> auf das Dokument zuzugreifen.
+mir.accesskey.setInfo.leading                   = Gegebenenfalls, besteht die M\u00F6glichkeit mit einem Zugriffsschl\u00FCssel \u00FCber folgenden
```

Comment:
I think the comma has to be different: "Gegebenenfalls besteht die M\u00F6glichkeit, ..."